src/dialogs/config-entry-system-options/dialog-config-entry-system-options.ts

```diff
@@ -54,27 +55,31 @@ class DialogConfigEntrySystemOptions extends LitElement {
     await this.updateComplete;
   }
 
+  public closeDialog(): void {
+    this._open = false;
+    this._error = "";
+    this._params = undefined;
+  }
+
   protected render(): TemplateResult {
     if (!this._params) {
       return html``;
     }
 
     return html`
-      <ha-paper-dialog
-        with-backdrop
-        opened
-        @opened-changed="${this._openedChanged}"
+      <ha-dialog
+        open=${this._open}
```

Comment:
We don't render it anyway when closed.
```suggestion
        open
```